Bottom: 1c48b13a7f6f0a8f24ba7ab90c4ea9d9a49ebffc
Top:    b334cf3155c0fe5c338a816f99ea1b26937096fa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 23:45:58 +0100

macro.h: Remove now unused SCREEN_SIGN_{X,Y} macros


---

diff --git a/src/macro.h b/src/macro.h
index 7fa0927..695d3b6 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -47,9 +47,6 @@
 #define	SWAP_X(x)		(SWAP_SIGN_X(x))
 #define	SWAP_Y(y)		(PCB->MaxHeight +SWAP_SIGN_Y(y))
 
-#define	TO_SCREEN_SIGN_X(x)	(SWAP_IDENT ? SWAP_SIGN_X(x) : (x))
-#define	TO_SCREEN_SIGN_Y(y)	(SWAP_IDENT ? SWAP_SIGN_Y(y) : (y))
-
 /* ---------------------------------------------------------------------------
  * misc macros, some might already be defined by <limits.h>
  */
